packages/arcgis-rest-request/src/request.ts

```diff
@@ -388,6 +388,7 @@ export function internalRequest(
         // encode the parameters into the query string
         const queryParams = encodeQueryString(params);
         // dont append a '?' unless parameters are actually present
+        /* istanbul ignore next */
         const urlWithQueryString =
           queryParams === "" ? url : url + "?" + encodeQueryString(params);
```

Comment:
this encodeQueryString seems to be happening twice, should we just append ...queryParams below instead of making the call again?
```suggestion
          queryParams === "" ? url : `${url}?${queryParams}`;
```
```suggestion
          queryParams === "" ? url : url + "?" + queryParams;
```